packages/vike-react-chakra/package.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "vike-react-chakra",
+    "version": "0.0.1",
+    "type": "module",
+    "exports": {
+        "./config": "./dist/integration/+config.js",
+        "./__internal/integration/Wrapper": "./dist/integration/Wrapper.js"
+    },
+    "scripts": {
+        "dev": "tsc --watch",
+        "build": "rimraf dist/ && tsc",
+        "release": "release-me patch",
+        "release:minor": "release-me minor",
+        "release:commit": "release-me commit"
+    },
+    "peerDependencies": {
+        "@chakra-ui/react": ">=3",
+        "@emotion/react": ">=11",
+        "react": ">=18.0.0",
+        "vike-react": ">=0.4.13"
+    },
+    "devDependencies": {
+        "@brillout/release-me": "0.3.8",
+        "@chakra-ui/react": "^3.0.2",
+        "@emotion/react": "^11.13.3",
+        "@types/node": "^20.11.17",
+        "@types/react": "^18.2.55",
+        "jsdom": "^24.0.0",
```

Comment:
That's surprising. Is it a peer dep? :thinking:

FYI the whole `vike-react-chakra` package will soon be ejectable, so polishing is extra worth it.

Reply:
`jsdom` ?, my bad, I'll remove it. As for `@types/node`, let me check if we need it here